src/mobile/ios/Argon2/Argon2IOS.m

```diff
@@ -11,7 +11,7 @@
 
 @interface RCT_EXTERN_MODULE(Argon2IOS, NSObject)
 // Export hash method to RN
-RCT_EXTERN_METHOD(hash:(NSString *)password salt:(NSString *)salt params:(NSDictionary *)params resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject);
+RCT_EXTERN_METHOD(hash:(NSArray *)password salt:(NSString *)salt params:(NSDictionary *)params resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject);
```

Comment:
We only want arrays of integers
```suggestion
RCT_EXTERN_METHOD(hash:(NSArray<NSInteger*>*)password salt:(NSString *)salt params:(NSDictionary *)params resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject);
```

Reply:
RCT Convert won't accept `NSArray<NSInteger*>*`, so I've changed it to `NSNumberArray`.